content/security/docs/runtime.md

```diff
@@ -32,7 +32,41 @@ AppArmor is similar to seccomp, only it restricts an container's capabilities in
+
+For more information, visit [Amazon GuardDuty User Guide - Configuring EKS Runtime Monitoring](https://docs.aws.amazon.com/guardduty/latest/ug/eks-protection-configuration.html)
+
+#### Enable Amazon GuardDuty EKS Audit Log Monitoring
+EKS Audit Log Monitoring helps you detect potentially suspicious activities in your EKS clusters within Amazon Elastic Kubernetes Service. When you enable EKS Audit Log Monitoring, GuardDuty immediately begins to monitor Kubernetes audit logs from your Amazon EKS clusters and analyze them for potentially malicious and suspicious activity. It consumes Kubernetes audit log events directly from the Amazon EKS control plane logging feature through an independent and duplicative stream of flow logs. This process does not require any additional set up or affect any existing Amazon EKS control plane logging configurations that you might have. When you disable EKS Audit Log Monitoring, GuardDuty immediately stops monitoring and analyzing the Kubernetes audit logs for your EKS resources. 
+
+For more information, visit [Amazon GuardDuty User Guide - EKS Audit Log Monitoring](https://docs.aws.amazon.com/guardduty/latest/ug/guardduty-eks-audit-log-monitoring.html). 
+
+#### Enable Amazon GuardDuty Malware Protection 
+Malware Protection helps you detect the potential presence of malware by scanning the Amazon Elastic Block Store (Amazon EBS) volumes that are attached to the Amazon Elastic Compute Cloud (Amazon EC2) instances and container workloads. Malware Protection provides scan options where you can decide if you want to include or exclude specific Amazon EC2 instances and container workloads at the time of scanning. It also provides an option to retain the snapshots of Amazon EBS volumes attached to the Amazon EC2 instances or container workloads, in your GuardDuty accounts. The snapshots get retained only when malware is found and Malware Protection findings are generated. Malware Protection offers two types of scans to detect potentially malicious activity in your Amazon EC2 instances and container workloads – GuardDuty-initiated malware scan and On-demand malware scan. 
+
+For more information, visit [Amazon GuardDuty User Guide - Malware Protection in Amazon GuardDuty (https://docs.aws.amazon.com/guardduty/latest/ug/malware-protection.html)
+
+#### Enable Amazon GuardDuty S3 Protection
```

Comment:
This is the EKS best practices guide. We are not including content for other AWS services. Please remove.